hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelineProvider.java

```diff
@@ -26,16 +26,20 @@
 
 import org.apache.hadoop.hdds.client.ReplicationConfig;
 import org.apache.hadoop.hdds.protocol.DatanodeDetails;
+import org.apache.hadoop.hdds.scm.SCMCommonPlacementPolicy;
 import org.apache.hadoop.hdds.scm.exceptions.SCMException;
 import org.apache.hadoop.hdds.scm.node.NodeManager;
 import org.apache.hadoop.hdds.scm.node.NodeStatus;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * Interface for creating pipelines.
  */
 public abstract class PipelineProvider<REPLICATION_CONFIG
     extends ReplicationConfig> {
-
+  private static final Logger LOG =
+      LoggerFactory.getLogger(SCMCommonPlacementPolicy.class);
```

Comment:
A copy-paste error? Normally we have the classname here.

Reply:
Oops yes I make this mistake all the time. Thanks.